src/snapred/backend/dao/reduction/ReductionRecord.py

```diff
@@ -17,35 +16,19 @@ class ReductionRecord(BaseModel):
     calibration, normalization, and pixel grouping details.
     """
 
-    runNumbers: List[str]
+    # Reduction has distinct registration attributes from Calibration and Normalization.
+    #   for this reason, this class is not derived from 'Record',
+    #   and does not include a 'CalculationParameters' instance.
     useLiteMode: bool
+    timestamp: float = Field(frozen=True, default=None)
+
+    # specific to reduction records
+    runNumber: str
     calibration: CalibrationRecord
     normalization: NormalizationRecord
     pixelGroupingParameters: Dict[str, List[PixelGroupingParameters]]
-    version: int
-    stateId: ObjectSHA
```

Comment:
We no longer need a state id within the the ReductionRecord? Does the indexor handle all of the "cataloging".

Reply:
It's just that the `runNumber` _implies_ the `stateId`, so including it would be redundant.

Reply:
Okay, that makes sense.